app/views/repositories/empty.html.erb

```diff
@@ -33,6 +33,7 @@ See doc/COPYRIGHT.rdoc for more details.
              message: l('repositories.errors.empty_repository')) %></p>
   </div>
 </div>
+<%= render partial: 'checkout_instructions', locals: { repository: @repository, instructions: @instructions } %>
```

Comment:
I'm open for discussions here, but I actually prefer it otherwise, as I parse the page top-down, and should receive the warning first.

![bildschirmfoto 2015-08-24 um 16 56 58](https://cloud.githubusercontent.com/assets/459462/9443418/34bba0c6-4a81-11e5-8589-1a797972440c.png)



![bildschirmfoto 2015-08-24 um 16 57 11](https://cloud.githubusercontent.com/assets/459462/9443419/34d393b6-4a81-11e5-98a5-c178412f6e49.png)